settingshelper.py

```diff
@@ -172,12 +172,19 @@ def celery_failure_handler(task, exc, task_id, args, kwargs, einfo):
 
 
 def get_allowed_websocket_channels(request, channels):
+    from django.conf import settings
     from django.core.exceptions import PermissionDenied
+    from corehq.apps.users.models import CouchUser
     if request.user and request.user.is_authenticated() and request.user.is_superuser:
         return channels
     else:
+        couch_user = CouchUser.from_django_user(request.user)
```

Comment:
in [the docs for the framework we use](https://django-websocket-redis.readthedocs.io/en/latest/usage.html#safety-considerations) it says that this function shouldn't perform any blocking requests, though it doesn't really explain why. I'm not sure if this only works in development mode, but i figured I could test on prod and then decide what to do.